backend/src/apiserver/model/run.go

```diff
@@ -18,6 +18,7 @@ type Run struct {
 	UUID               string `gorm:"column:UUID; not null; primary_key"`
 	DisplayName        string `gorm:"column:DisplayName; not null;"` /* The name that user provides. Can contain special characters*/
 	Name               string `gorm:"column:Name; not null;"`        /* The name of the K8s resource. Follow regex '[a-z0-9]([-a-z0-9]*[a-z0-9])?'*/
+	StorageState       string `gorm:"column:StorageState;"`
```

Comment:
Should StorageState have a default value? Maybe we can also add not null, and enforce that all runs are in a valid state (archived or available). 

Can we add a test to the CreateRun call to ensure that the state is valid?

Reply:
Makes sense, done.
Didn't add a new test, just a field to the expected object in old tests, since it's required now.

Reply:
Btw, this was done in the [insertion code path here](https://github.com/kubeflow/pipelines/pull/552/files#diff-b5ccfe10b993bfe8c4b20aabb6ed8d66R263). Let me know if you think there's a better way. @IronPan @neuromage.